python/py-hypothesis/Portfile

```diff
@@ -50,6 +50,7 @@ if {${name} ne ${subport}} {
     }
 
     if {${python.version} == 35} {
```

Comment:
Do we actually have a good reason to support ancient Python versions for this, or could we just get rid of that?

Reply:
I typically do greps to see if it is required anywhere before removing.  I haven't had time yet.